release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,18 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-column-management"]
+==== Column management
+
+A *Manage columns* icon image:manage-columns.png[title="Manage Columns icon"] is
+now added to resources pages. When you click on the icon, all column names are
+listed with check boxes. Deselecting a check box will remove that column from
+the table view. Selecting a check box will add that column to the table view.
+Clicking *Save* will save the changes that you make. Clicking *Restore default
```

Comment:
A maximum of 9 columns from both sides of the modal are available for display at one time.